variables.tf

```diff
@@ -57,7 +57,7 @@ variable "sls_service_name" {
 }
 
 variable "lambda_role_name" {
-  description = "Custom Lambda role to override the default Serverless one. The custom role should provide at least the same level of access as the default."
+  description = "Name of a custom Lambda role to override the default Serverless one. The custom role should provide at least the same level of access as the default."
```

Comment:
We should add what our "dynamic default" is to the `description` as although our real `default` is `""` we do provide one in our `local` gathering. Suggest something like:

```
 Defaults to: `tf-SERVICE_NAME-STAGE-lambda-execution`.
```